### PR TITLE
Replace receiver with celery

### PR DESCRIPTION
**Pull request by Fagtoy**
replaced sender/receiver with celery tasks
**NOTE**: should be merged at the same time with this PR: https://github.com/YangCatalog/deployment/pull/192

### CODE REVIEW

File: jobs/jobs_information.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+def get_response(celery_app_instance: BackendCeleryApp, job_id: str) -> tuple[str, str]:
+    """
+    Get response according to job_id. It can be either 'Failed', 'In progress', 'Finished successfully'
+
+    Arguments:
+        :param celery_app_instance (BackendCeleryApp) instance of the celery app
+        :param job_id: (str) id of the celery task
+    :return (tuple[str, str]) Returns the job status which can one of the following -
```
Comment (richardzilincikPantheon) [inline jobs/jobs_information.py:31]:
```suggestion
    :return (tuple[str, str]) Returns the job status which can be one of the following -
```

**Review by richardzilincikPantheon — CHANGES_REQUESTED** [review]
No important, but is there any advantage to using `.s().apply_async()` over `.delay()`?

File: api/views/health_check.py [hunk 2]
```diff
@@ -538,14 +505,47 @@ def health_check_yangcatalog():
...
+    status = StatusMessage.IN_PROGRESS.value
+    reason = ''
+    while attempts > 0:
+        status, reason = get_response(app_config.celery_app, result.id)
+        if status == StatusMessage.IN_PROGRESS:
+            time.sleep(5)
+            attempts -= 1
+            continue
+        break
+    message_mapping = {
```
[the remark this answers is not in the document]
Comment (Fagtoy) [reply api/views/health_check.py:529]: Unfortunately, this status isn't helpful for us, as far as there's only one task can be run at a time, while one long-running task is being executed, other tasks in the queue will be in the PENDING state, so there's no difference for us if it's this task in PENDING or STARTED status, we assume that this task is "In Progress"

**Comment by Fagtoy** [comment]
> No important, but is there any advantage to using `.s().apply_async()` over `.delay()`?

there's no advantage for now but if in the future the project needs more flexibility (different queues or some other configurations), then it's better to use apply_async, so I decided to use it now, so we assume it as the default calling method for the future

**Review by richardzilincikPantheon — APPROVED** [review]

**Pull request merged**